DEBUG - 2025-06-29 21:13:10 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 21:16:08 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 21:16:09 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-06-29 21:19:53 --> mysqli_sql_exception: Failed to open the referenced table 'formtemplate' in /Users/mehmetaydin/docker/server/src/projexiaa/system/Database/MySQLi/Connection.php:327
Stack trace:
#0 /Users/mehmetaydin/docker/server/src/projexiaa/system/Database/MySQLi/Connection.php(327): mysqli->query('CREATE TABLE `f...', 0)
#1 /Users/mehmetaydin/docker/server/src/projexiaa/system/Database/BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `f...')
#2 /Users/mehmetaydin/docker/server/src/projexiaa/system/Database/BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `f...')
#3 /Users/mehmetaydin/docker/server/src/projexiaa/system/Database/Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `f...')
#4 /Users/mehmetaydin/docker/server/src/projexiaa/app/Database/Migrations/2025-06-14-063713_CreateFormTemplates.php(41): CodeIgniter\Database\Forge->createTable('formtemplate_va...')
#5 /Users/mehmetaydin/docker/server/src/projexiaa/system/Database/MigrationRunner.php(876): App\Database\Migrations\CreateFormTemplates->up()
#6 /Users/mehmetaydin/docker/server/src/projexiaa/system/Database/MigrationRunner.php(182): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 /Users/mehmetaydin/docker/server/src/projexiaa/system/Commands/Database/Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 /Users/mehmetaydin/docker/server/src/projexiaa/system/CLI/Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 /Users/mehmetaydin/docker/server/src/projexiaa/system/CLI/Console.php(47): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 /Users/mehmetaydin/docker/server/src/projexiaa/system/Boot.php(360): CodeIgniter\CLI\Console->run()
#11 /Users/mehmetaydin/docker/server/src/projexiaa/system/Boot.php(104): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 /Users/mehmetaydin/docker/server/src/projexiaa/spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-06-29 21:21:50 --> mysqli_sql_exception: Failed to open the referenced table 'formtemplate' in /Users/mehmetaydin/docker/server/src/projexiaa/system/Database/MySQLi/Connection.php:327
Stack trace:
#0 /Users/mehmetaydin/docker/server/src/projexiaa/system/Database/MySQLi/Connection.php(327): mysqli->query('CREATE TABLE `f...', 0)
#1 /Users/mehmetaydin/docker/server/src/projexiaa/system/Database/BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `f...')
#2 /Users/mehmetaydin/docker/server/src/projexiaa/system/Database/BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `f...')
#3 /Users/mehmetaydin/docker/server/src/projexiaa/system/Database/Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `f...')
#4 /Users/mehmetaydin/docker/server/src/projexiaa/app/Database/Migrations/2025-06-14-063713_CreateFormTemplates.php(41): CodeIgniter\Database\Forge->createTable('formtemplate_va...')
#5 /Users/mehmetaydin/docker/server/src/projexiaa/system/Database/MigrationRunner.php(876): App\Database\Migrations\CreateFormTemplates->up()
#6 /Users/mehmetaydin/docker/server/src/projexiaa/system/Database/MigrationRunner.php(182): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 /Users/mehmetaydin/docker/server/src/projexiaa/system/Commands/Database/Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 /Users/mehmetaydin/docker/server/src/projexiaa/system/CLI/Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 /Users/mehmetaydin/docker/server/src/projexiaa/system/CLI/Console.php(47): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 /Users/mehmetaydin/docker/server/src/projexiaa/system/Boot.php(360): CodeIgniter\CLI\Console->run()
#11 /Users/mehmetaydin/docker/server/src/projexiaa/system/Boot.php(104): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 /Users/mehmetaydin/docker/server/src/projexiaa/spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-06-29 21:30:33 --> Error connecting to the database: mysqli_sql_exception: Connection refused in /var/www/html/projexiaa/system/Database/MySQLi/Connection.php:201
Stack trace:
#0 /var/www/html/projexiaa/system/Database/MySQLi/Connection.php(201): mysqli->real_connect('127.0.0.1', 'root', Object(SensitiveParameterValue), 'projexia', 3306, '', 0)
#1 /var/www/html/projexiaa/system/Database/BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 /var/www/html/projexiaa/system/Database/BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#3 /var/www/html/projexiaa/system/Database/BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 /var/www/html/projexiaa/system/Validation/Rules.php(175): CodeIgniter\Database\BaseBuilder->get()
#5 /var/www/html/projexiaa/system/Validation/StrictRules/Rules.php(187): CodeIgniter\Validation\Rules->is_unique('5435781808', 'users.user_phon...', Array)
#6 /var/www/html/projexiaa/system/Validation/Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('5435781808', 'users.user_phon...', Array, NULL, 'user_phone')
#7 /var/www/html/projexiaa/system/Validation/Validation.php(201): CodeIgniter\Validation\Validation->processRules('user_phone', 'user_phone', '5435781808', Array, Array, 'user_phone')
#8 /var/www/html/projexiaa/app/Controllers/RegisterController.php(46): CodeIgniter\Validation\Validation->run()
#9 /var/www/html/projexiaa/system/CodeIgniter.php(933): App\Controllers\RegisterController->submit()
#10 /var/www/html/projexiaa/system/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
#11 /var/www/html/projexiaa/system/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 /var/www/html/projexiaa/system/Boot.php(334): CodeIgniter\CodeIgniter->run()
#13 /var/www/html/projexiaa/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 /var/www/html/projexiaa/index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Connection refused in /var/www/html/projexiaa/system/Database/MySQLi/Connection.php:246
Stack trace:
#0 /var/www/html/projexiaa/system/Database/BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 /var/www/html/projexiaa/system/Database/BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#2 /var/www/html/projexiaa/system/Database/BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#3 /var/www/html/projexiaa/system/Validation/Rules.php(175): CodeIgniter\Database\BaseBuilder->get()
#4 /var/www/html/projexiaa/system/Validation/StrictRules/Rules.php(187): CodeIgniter\Validation\Rules->is_unique('5435781808', 'users.user_phon...', Array)
#5 /var/www/html/projexiaa/system/Validation/Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('5435781808', 'users.user_phon...', Array, NULL, 'user_phone')
#6 /var/www/html/projexiaa/system/Validation/Validation.php(201): CodeIgniter\Validation\Validation->processRules('user_phone', 'user_phone', '5435781808', Array, Array, 'user_phone')
#7 /var/www/html/projexiaa/app/Controllers/RegisterController.php(46): CodeIgniter\Validation\Validation->run()
#8 /var/www/html/projexiaa/system/CodeIgniter.php(933): App\Controllers\RegisterController->submit()
#9 /var/www/html/projexiaa/system/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
#10 /var/www/html/projexiaa/system/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 /var/www/html/projexiaa/system/Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 /var/www/html/projexiaa/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 /var/www/html/projexiaa/index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2025-06-29 21:30:33 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Connection refused
[Method: POST, Route: register/submit]
in SYSTEMPATH/Database/BaseConnection.php on line 467.
 1 SYSTEMPATH/Database/BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH/Database/BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `users`
WHERE `user_phone` = :user_phone:
 LIMIT 1', [...], false)
 3 SYSTEMPATH/Validation/Rules.php(175): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH/Validation/StrictRules/Rules.php(187): CodeIgniter\Validation\Rules->is_unique('5435781808', 'users.user_phone', [...])
 5 SYSTEMPATH/Validation/Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('5435781808', 'users.user_phone', [...], null, 'user_phone')
 6 SYSTEMPATH/Validation/Validation.php(201): CodeIgniter\Validation\Validation->processRules('user_phone', 'user_phone', '5435781808', [...], [...], 'user_phone')
 7 APPPATH/Controllers/RegisterController.php(46): CodeIgniter\Validation\Validation->run()
 8 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\RegisterController->submit()
 9 SYSTEMPATH/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
10 SYSTEMPATH/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH/Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH/index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-06-29 21:30:39 --> Error connecting to the database: mysqli_sql_exception: Connection refused in /var/www/html/projexiaa/system/Database/MySQLi/Connection.php:201
Stack trace:
#0 /var/www/html/projexiaa/system/Database/MySQLi/Connection.php(201): mysqli->real_connect('127.0.0.1', 'root', Object(SensitiveParameterValue), 'projexia', 3306, '', 0)
#1 /var/www/html/projexiaa/system/Database/BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 /var/www/html/projexiaa/system/Database/BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#3 /var/www/html/projexiaa/system/Database/BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 /var/www/html/projexiaa/system/Validation/Rules.php(175): CodeIgniter\Database\BaseBuilder->get()
#5 /var/www/html/projexiaa/system/Validation/StrictRules/Rules.php(187): CodeIgniter\Validation\Rules->is_unique('5435781808', 'users.user_phon...', Array)
#6 /var/www/html/projexiaa/system/Validation/Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('5435781808', 'users.user_phon...', Array, NULL, 'user_phone')
#7 /var/www/html/projexiaa/system/Validation/Validation.php(201): CodeIgniter\Validation\Validation->processRules('user_phone', 'user_phone', '5435781808', Array, Array, 'user_phone')
#8 /var/www/html/projexiaa/app/Controllers/RegisterController.php(46): CodeIgniter\Validation\Validation->run()
#9 /var/www/html/projexiaa/system/CodeIgniter.php(933): App\Controllers\RegisterController->submit()
#10 /var/www/html/projexiaa/system/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
#11 /var/www/html/projexiaa/system/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 /var/www/html/projexiaa/system/Boot.php(334): CodeIgniter\CodeIgniter->run()
#13 /var/www/html/projexiaa/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 /var/www/html/projexiaa/index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Connection refused in /var/www/html/projexiaa/system/Database/MySQLi/Connection.php:246
Stack trace:
#0 /var/www/html/projexiaa/system/Database/BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 /var/www/html/projexiaa/system/Database/BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#2 /var/www/html/projexiaa/system/Database/BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#3 /var/www/html/projexiaa/system/Validation/Rules.php(175): CodeIgniter\Database\BaseBuilder->get()
#4 /var/www/html/projexiaa/system/Validation/StrictRules/Rules.php(187): CodeIgniter\Validation\Rules->is_unique('5435781808', 'users.user_phon...', Array)
#5 /var/www/html/projexiaa/system/Validation/Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('5435781808', 'users.user_phon...', Array, NULL, 'user_phone')
#6 /var/www/html/projexiaa/system/Validation/Validation.php(201): CodeIgniter\Validation\Validation->processRules('user_phone', 'user_phone', '5435781808', Array, Array, 'user_phone')
#7 /var/www/html/projexiaa/app/Controllers/RegisterController.php(46): CodeIgniter\Validation\Validation->run()
#8 /var/www/html/projexiaa/system/CodeIgniter.php(933): App\Controllers\RegisterController->submit()
#9 /var/www/html/projexiaa/system/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
#10 /var/www/html/projexiaa/system/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 /var/www/html/projexiaa/system/Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 /var/www/html/projexiaa/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 /var/www/html/projexiaa/index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2025-06-29 21:30:39 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Connection refused
[Method: POST, Route: register/submit]
in SYSTEMPATH/Database/BaseConnection.php on line 467.
 1 SYSTEMPATH/Database/BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH/Database/BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `users`
WHERE `user_phone` = :user_phone:
 LIMIT 1', [...], false)
 3 SYSTEMPATH/Validation/Rules.php(175): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH/Validation/StrictRules/Rules.php(187): CodeIgniter\Validation\Rules->is_unique('5435781808', 'users.user_phone', [...])
 5 SYSTEMPATH/Validation/Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('5435781808', 'users.user_phone', [...], null, 'user_phone')
 6 SYSTEMPATH/Validation/Validation.php(201): CodeIgniter\Validation\Validation->processRules('user_phone', 'user_phone', '5435781808', [...], [...], 'user_phone')
 7 APPPATH/Controllers/RegisterController.php(46): CodeIgniter\Validation\Validation->run()
 8 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\RegisterController->submit()
 9 SYSTEMPATH/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
10 SYSTEMPATH/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH/Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH/index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-06-29 21:30:43 --> Error connecting to the database: mysqli_sql_exception: Connection refused in /var/www/html/projexiaa/system/Database/MySQLi/Connection.php:201
Stack trace:
#0 /var/www/html/projexiaa/system/Database/MySQLi/Connection.php(201): mysqli->real_connect('127.0.0.1', 'root', Object(SensitiveParameterValue), 'projexia', 3306, '', 0)
#1 /var/www/html/projexiaa/system/Database/BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 /var/www/html/projexiaa/system/Database/BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#3 /var/www/html/projexiaa/system/Database/BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 /var/www/html/projexiaa/system/Validation/Rules.php(175): CodeIgniter\Database\BaseBuilder->get()
#5 /var/www/html/projexiaa/system/Validation/StrictRules/Rules.php(187): CodeIgniter\Validation\Rules->is_unique('5435781808', 'users.user_phon...', Array)
#6 /var/www/html/projexiaa/system/Validation/Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('5435781808', 'users.user_phon...', Array, NULL, 'user_phone')
#7 /var/www/html/projexiaa/system/Validation/Validation.php(201): CodeIgniter\Validation\Validation->processRules('user_phone', 'user_phone', '5435781808', Array, Array, 'user_phone')
#8 /var/www/html/projexiaa/app/Controllers/RegisterController.php(46): CodeIgniter\Validation\Validation->run()
#9 /var/www/html/projexiaa/system/CodeIgniter.php(933): App\Controllers\RegisterController->submit()
#10 /var/www/html/projexiaa/system/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
#11 /var/www/html/projexiaa/system/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 /var/www/html/projexiaa/system/Boot.php(334): CodeIgniter\CodeIgniter->run()
#13 /var/www/html/projexiaa/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 /var/www/html/projexiaa/index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Connection refused in /var/www/html/projexiaa/system/Database/MySQLi/Connection.php:246
Stack trace:
#0 /var/www/html/projexiaa/system/Database/BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 /var/www/html/projexiaa/system/Database/BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#2 /var/www/html/projexiaa/system/Database/BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#3 /var/www/html/projexiaa/system/Validation/Rules.php(175): CodeIgniter\Database\BaseBuilder->get()
#4 /var/www/html/projexiaa/system/Validation/StrictRules/Rules.php(187): CodeIgniter\Validation\Rules->is_unique('5435781808', 'users.user_phon...', Array)
#5 /var/www/html/projexiaa/system/Validation/Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('5435781808', 'users.user_phon...', Array, NULL, 'user_phone')
#6 /var/www/html/projexiaa/system/Validation/Validation.php(201): CodeIgniter\Validation\Validation->processRules('user_phone', 'user_phone', '5435781808', Array, Array, 'user_phone')
#7 /var/www/html/projexiaa/app/Controllers/RegisterController.php(46): CodeIgniter\Validation\Validation->run()
#8 /var/www/html/projexiaa/system/CodeIgniter.php(933): App\Controllers\RegisterController->submit()
#9 /var/www/html/projexiaa/system/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
#10 /var/www/html/projexiaa/system/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 /var/www/html/projexiaa/system/Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 /var/www/html/projexiaa/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 /var/www/html/projexiaa/index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2025-06-29 21:30:43 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Connection refused
[Method: POST, Route: register/submit]
in SYSTEMPATH/Database/BaseConnection.php on line 467.
 1 SYSTEMPATH/Database/BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH/Database/BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `users`
WHERE `user_phone` = :user_phone:
 LIMIT 1', [...], false)
 3 SYSTEMPATH/Validation/Rules.php(175): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH/Validation/StrictRules/Rules.php(187): CodeIgniter\Validation\Rules->is_unique('5435781808', 'users.user_phone', [...])
 5 SYSTEMPATH/Validation/Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('5435781808', 'users.user_phone', [...], null, 'user_phone')
 6 SYSTEMPATH/Validation/Validation.php(201): CodeIgniter\Validation\Validation->processRules('user_phone', 'user_phone', '5435781808', [...], [...], 'user_phone')
 7 APPPATH/Controllers/RegisterController.php(46): CodeIgniter\Validation\Validation->run()
 8 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\RegisterController->submit()
 9 SYSTEMPATH/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
10 SYSTEMPATH/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH/Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH/index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-06-29 21:31:28 --> Error connecting to the database: mysqli_sql_exception: Connection refused in /var/www/html/projexiaa/system/Database/MySQLi/Connection.php:201
Stack trace:
#0 /var/www/html/projexiaa/system/Database/MySQLi/Connection.php(201): mysqli->real_connect('127.0.0.1', 'root', Object(SensitiveParameterValue), 'projexia', 3306, '', 0)
#1 /var/www/html/projexiaa/system/Database/BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 /var/www/html/projexiaa/system/Database/BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#3 /var/www/html/projexiaa/system/Database/BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 /var/www/html/projexiaa/system/Validation/Rules.php(175): CodeIgniter\Database\BaseBuilder->get()
#5 /var/www/html/projexiaa/system/Validation/StrictRules/Rules.php(187): CodeIgniter\Validation\Rules->is_unique('5555435785', 'users.user_phon...', Array)
#6 /var/www/html/projexiaa/system/Validation/Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('5555435785', 'users.user_phon...', Array, NULL, 'user_phone')
#7 /var/www/html/projexiaa/system/Validation/Validation.php(201): CodeIgniter\Validation\Validation->processRules('user_phone', 'user_phone', '5555435785', Array, Array, 'user_phone')
#8 /var/www/html/projexiaa/app/Controllers/RegisterController.php(46): CodeIgniter\Validation\Validation->run()
#9 /var/www/html/projexiaa/system/CodeIgniter.php(933): App\Controllers\RegisterController->submit()
#10 /var/www/html/projexiaa/system/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
#11 /var/www/html/projexiaa/system/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 /var/www/html/projexiaa/system/Boot.php(334): CodeIgniter\CodeIgniter->run()
#13 /var/www/html/projexiaa/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 /var/www/html/projexiaa/index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Connection refused in /var/www/html/projexiaa/system/Database/MySQLi/Connection.php:246
Stack trace:
#0 /var/www/html/projexiaa/system/Database/BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 /var/www/html/projexiaa/system/Database/BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#2 /var/www/html/projexiaa/system/Database/BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#3 /var/www/html/projexiaa/system/Validation/Rules.php(175): CodeIgniter\Database\BaseBuilder->get()
#4 /var/www/html/projexiaa/system/Validation/StrictRules/Rules.php(187): CodeIgniter\Validation\Rules->is_unique('5555435785', 'users.user_phon...', Array)
#5 /var/www/html/projexiaa/system/Validation/Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('5555435785', 'users.user_phon...', Array, NULL, 'user_phone')
#6 /var/www/html/projexiaa/system/Validation/Validation.php(201): CodeIgniter\Validation\Validation->processRules('user_phone', 'user_phone', '5555435785', Array, Array, 'user_phone')
#7 /var/www/html/projexiaa/app/Controllers/RegisterController.php(46): CodeIgniter\Validation\Validation->run()
#8 /var/www/html/projexiaa/system/CodeIgniter.php(933): App\Controllers\RegisterController->submit()
#9 /var/www/html/projexiaa/system/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
#10 /var/www/html/projexiaa/system/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 /var/www/html/projexiaa/system/Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 /var/www/html/projexiaa/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 /var/www/html/projexiaa/index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2025-06-29 21:31:28 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Connection refused
[Method: POST, Route: register/submit]
in SYSTEMPATH/Database/BaseConnection.php on line 467.
 1 SYSTEMPATH/Database/BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH/Database/BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `users`
WHERE `user_phone` = :user_phone:
 LIMIT 1', [...], false)
 3 SYSTEMPATH/Validation/Rules.php(175): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH/Validation/StrictRules/Rules.php(187): CodeIgniter\Validation\Rules->is_unique('5555435785', 'users.user_phone', [...])
 5 SYSTEMPATH/Validation/Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('5555435785', 'users.user_phone', [...], null, 'user_phone')
 6 SYSTEMPATH/Validation/Validation.php(201): CodeIgniter\Validation\Validation->processRules('user_phone', 'user_phone', '5555435785', [...], [...], 'user_phone')
 7 APPPATH/Controllers/RegisterController.php(46): CodeIgniter\Validation\Validation->run()
 8 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\RegisterController->submit()
 9 SYSTEMPATH/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
10 SYSTEMPATH/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH/Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH/index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-06-29 21:32:40 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 21:32:42 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-06-29 21:32:52 --> mysqli_sql_exception: Table 'projexia.user_roles' doesn't exist in /var/www/html/projexiaa/system/Database/MySQLi/Connection.php:327
Stack trace:
#0 /var/www/html/projexiaa/system/Database/MySQLi/Connection.php(327): mysqli->query('SELECT `user_ro...', 0)
#1 /var/www/html/projexiaa/system/Database/BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `user_ro...')
#2 /var/www/html/projexiaa/system/Database/BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `user_ro...')
#3 /var/www/html/projexiaa/system/Database/BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `user_ro...', Array, false)
#4 /var/www/html/projexiaa/app/Models/LoginModel.php(54): CodeIgniter\Database\BaseBuilder->get()
#5 /var/www/html/projexiaa/app/Controllers/LoginController.php(27): App\Models\LoginModel->getUserRoleIn('1')
#6 /var/www/html/projexiaa/system/CodeIgniter.php(933): App\Controllers\LoginController->loginActions()
#7 /var/www/html/projexiaa/system/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\LoginController))
#8 /var/www/html/projexiaa/system/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 /var/www/html/projexiaa/system/Boot.php(334): CodeIgniter\CodeIgniter->run()
#10 /var/www/html/projexiaa/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 /var/www/html/projexiaa/index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2025-06-29 21:32:52 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'projexia.user_roles' doesn't exist
[Method: POST, Route: login]
in SYSTEMPATH/Database/BaseConnection.php on line 692.
 1 SYSTEMPATH/Database/BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `user_roles`.`role_id`, `roles`.`role_name`
FROM `user_roles`
JOIN `roles` ON `roles`.`id` = `user_roles`.`role_id`
WHERE `user_roles`.`user_id` = :user_roles.user_id:', [...], false)
 2 APPPATH/Models/LoginModel.php(54): CodeIgniter\Database\BaseBuilder->get()
 3 APPPATH/Controllers/LoginController.php(27): App\Models\LoginModel->getUserRoleIn('1')
 4 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\LoginController->loginActions()
 5 SYSTEMPATH/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\LoginController))
 6 SYSTEMPATH/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH/Boot.php(334): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH/index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-29 21:32:52 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'projexia.user_roles' doesn't exist
in SYSTEMPATH/Database/MySQLi/Connection.php on line 332.
 1 SYSTEMPATH/Database/BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `user_roles`.`role_id`, `roles`.`role_name`
FROM `user_roles`
JOIN `roles` ON `roles`.`id` = `user_roles`.`role_id`
WHERE `user_roles`.`user_id` = \'1\'')
 2 SYSTEMPATH/Database/BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `user_roles`.`role_id`, `roles`.`role_name`
FROM `user_roles`
JOIN `roles` ON `roles`.`id` = `user_roles`.`role_id`
WHERE `user_roles`.`user_id` = \'1\'')
 3 SYSTEMPATH/Database/BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `user_roles`.`role_id`, `roles`.`role_name`
FROM `user_roles`
JOIN `roles` ON `roles`.`id` = `user_roles`.`role_id`
WHERE `user_roles`.`user_id` = :user_roles.user_id:', [...], false)
 4 APPPATH/Models/LoginModel.php(54): CodeIgniter\Database\BaseBuilder->get()
 5 APPPATH/Controllers/LoginController.php(27): App\Models\LoginModel->getUserRoleIn('1')
 6 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\LoginController->loginActions()
 7 SYSTEMPATH/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\LoginController))
 8 SYSTEMPATH/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH/Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH/index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-29 21:32:52 --> [Caused by] mysqli_sql_exception: Table 'projexia.user_roles' doesn't exist
in SYSTEMPATH/Database/MySQLi/Connection.php on line 327.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(327): mysqli->query('SELECT `user_roles`.`role_id`, `roles`.`role_name`
FROM `user_roles`
JOIN `roles` ON `roles`.`id` = `user_roles`.`role_id`
WHERE `user_roles`.`user_id` = \'1\'', 0)
 2 SYSTEMPATH/Database/BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `user_roles`.`role_id`, `roles`.`role_name`
FROM `user_roles`
JOIN `roles` ON `roles`.`id` = `user_roles`.`role_id`
WHERE `user_roles`.`user_id` = \'1\'')
 3 SYSTEMPATH/Database/BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `user_roles`.`role_id`, `roles`.`role_name`
FROM `user_roles`
JOIN `roles` ON `roles`.`id` = `user_roles`.`role_id`
WHERE `user_roles`.`user_id` = \'1\'')
 4 SYSTEMPATH/Database/BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `user_roles`.`role_id`, `roles`.`role_name`
FROM `user_roles`
JOIN `roles` ON `roles`.`id` = `user_roles`.`role_id`
WHERE `user_roles`.`user_id` = :user_roles.user_id:', [...], false)
 5 APPPATH/Models/LoginModel.php(54): CodeIgniter\Database\BaseBuilder->get()
 6 APPPATH/Controllers/LoginController.php(27): App\Models\LoginModel->getUserRoleIn('1')
 7 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\LoginController->loginActions()
 8 SYSTEMPATH/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\LoginController))
 9 SYSTEMPATH/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH/Boot.php(334): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH/index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-06-29 21:33:14 --> mysqli_sql_exception: Failed to open the referenced table 'formtemplate' in /Users/mehmetaydin/docker/server/src/projexiaa/system/Database/MySQLi/Connection.php:327
Stack trace:
#0 /Users/mehmetaydin/docker/server/src/projexiaa/system/Database/MySQLi/Connection.php(327): mysqli->query('CREATE TABLE `f...', 0)
#1 /Users/mehmetaydin/docker/server/src/projexiaa/system/Database/BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `f...')
#2 /Users/mehmetaydin/docker/server/src/projexiaa/system/Database/BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `f...')
#3 /Users/mehmetaydin/docker/server/src/projexiaa/system/Database/Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `f...')
#4 /Users/mehmetaydin/docker/server/src/projexiaa/app/Database/Migrations/2025-06-14-063713_CreateFormTemplates.php(41): CodeIgniter\Database\Forge->createTable('formtemplate_va...')
#5 /Users/mehmetaydin/docker/server/src/projexiaa/system/Database/MigrationRunner.php(876): App\Database\Migrations\CreateFormTemplates->up()
#6 /Users/mehmetaydin/docker/server/src/projexiaa/system/Database/MigrationRunner.php(182): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 /Users/mehmetaydin/docker/server/src/projexiaa/system/Commands/Database/Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 /Users/mehmetaydin/docker/server/src/projexiaa/system/CLI/Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 /Users/mehmetaydin/docker/server/src/projexiaa/system/CLI/Console.php(47): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 /Users/mehmetaydin/docker/server/src/projexiaa/system/Boot.php(360): CodeIgniter\CLI\Console->run()
#11 /Users/mehmetaydin/docker/server/src/projexiaa/system/Boot.php(104): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 /Users/mehmetaydin/docker/server/src/projexiaa/spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-06-29 21:39:06 --> Error connecting to the database: mysqli_sql_exception: Connection refused in /var/www/html/projexiaa/system/Database/MySQLi/Connection.php:201
Stack trace:
#0 /var/www/html/projexiaa/system/Database/MySQLi/Connection.php(201): mysqli->real_connect('127.0.0.1', 'root', Object(SensitiveParameterValue), 'projexia', 3306, '', 0)
#1 /var/www/html/projexiaa/system/Database/BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 /var/www/html/projexiaa/system/Database/BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#3 /var/www/html/projexiaa/system/Database/BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 /var/www/html/projexiaa/app/Models/LoginModel.php(33): CodeIgniter\Database\BaseBuilder->get()
#5 /var/www/html/projexiaa/app/Controllers/LoginController.php(22): App\Models\LoginModel->getUser('cnd@gmail.com')
#6 /var/www/html/projexiaa/system/CodeIgniter.php(933): App\Controllers\LoginController->loginActions()
#7 /var/www/html/projexiaa/system/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\LoginController))
#8 /var/www/html/projexiaa/system/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 /var/www/html/projexiaa/system/Boot.php(334): CodeIgniter\CodeIgniter->run()
#10 /var/www/html/projexiaa/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 /var/www/html/projexiaa/index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Connection refused in /var/www/html/projexiaa/system/Database/MySQLi/Connection.php:246
Stack trace:
#0 /var/www/html/projexiaa/system/Database/BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 /var/www/html/projexiaa/system/Database/BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#2 /var/www/html/projexiaa/system/Database/BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 /var/www/html/projexiaa/app/Models/LoginModel.php(33): CodeIgniter\Database\BaseBuilder->get()
#4 /var/www/html/projexiaa/app/Controllers/LoginController.php(22): App\Models\LoginModel->getUser('cnd@gmail.com')
#5 /var/www/html/projexiaa/system/CodeIgniter.php(933): App\Controllers\LoginController->loginActions()
#6 /var/www/html/projexiaa/system/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\LoginController))
#7 /var/www/html/projexiaa/system/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 /var/www/html/projexiaa/system/Boot.php(334): CodeIgniter\CodeIgniter->run()
#9 /var/www/html/projexiaa/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 /var/www/html/projexiaa/index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 {main}
CRITICAL - 2025-06-29 21:39:06 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Connection refused
[Method: POST, Route: login]
in SYSTEMPATH/Database/BaseConnection.php on line 467.
 1 SYSTEMPATH/Database/BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH/Database/BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`
WHERE `user_mail` = :user_mail:', [...], false)
 3 APPPATH/Models/LoginModel.php(33): CodeIgniter\Database\BaseBuilder->get()
 4 APPPATH/Controllers/LoginController.php(22): App\Models\LoginModel->getUser('cnd@gmail.com')
 5 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\LoginController->loginActions()
 6 SYSTEMPATH/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\LoginController))
 7 SYSTEMPATH/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH/Boot.php(334): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH/index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-06-29 21:39:24 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 21:39:24 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 21:39:35 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 21:39:35 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 21:39:54 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 21:39:56 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 21:40:04 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 21:40:05 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 21:43:02 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 21:43:49 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 21:43:50 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 21:45:00 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 21:45:01 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-06-29 21:45:22 --> mysqli_sql_exception: Duplicate entry '' for key 'users.user_username' in /var/www/html/projexiaa/system/Database/MySQLi/Connection.php:327
Stack trace:
#0 /var/www/html/projexiaa/system/Database/MySQLi/Connection.php(327): mysqli->query('INSERT INTO `us...', 0)
#1 /var/www/html/projexiaa/system/Database/BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 /var/www/html/projexiaa/system/Database/BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 /var/www/html/projexiaa/system/Database/BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 /var/www/html/projexiaa/system/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 /var/www/html/projexiaa/system/BaseModel.php(843): CodeIgniter\Model->doInsert(Array)
#6 /var/www/html/projexiaa/system/Model.php(800): CodeIgniter\BaseModel->insert(Array, true)
#7 /var/www/html/projexiaa/app/Models/LoginModel.php(45): CodeIgniter\Model->insert(Array)
#8 /var/www/html/projexiaa/app/Controllers/RegisterController.php(74): App\Models\LoginModel->insertUser(Array)
#9 /var/www/html/projexiaa/system/CodeIgniter.php(933): App\Controllers\RegisterController->submit()
#10 /var/www/html/projexiaa/system/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
#11 /var/www/html/projexiaa/system/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 /var/www/html/projexiaa/system/Boot.php(334): CodeIgniter\CodeIgniter->run()
#13 /var/www/html/projexiaa/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 /var/www/html/projexiaa/index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}
CRITICAL - 2025-06-29 21:45:22 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '' for key 'users.user_username'
[Method: POST, Route: register/submit]
in SYSTEMPATH/Database/BaseConnection.php on line 692.
 1 SYSTEMPATH/Database/BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`user_institution`, `user_gender`, `user_appellation`, `user_name`, `user_phone`, `user_orcid`, `user_mail`, `user_password`, `created_at`, `updated_at`) VALUES (:user_institution:, :user_gender:, :user_appellation:, :user_name:, :user_phone:, :user_orcid:, :user_mail:, :user_password:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH/BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH/Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH/Models/LoginModel.php(45): CodeIgniter\Model->insert([...])
 6 APPPATH/Controllers/RegisterController.php(74): App\Models\LoginModel->insertUser([...])
 7 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\RegisterController->submit()
 8 SYSTEMPATH/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
 9 SYSTEMPATH/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH/Boot.php(334): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH/index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-29 21:45:22 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '' for key 'users.user_username'
in SYSTEMPATH/Database/MySQLi/Connection.php on line 332.
 1 SYSTEMPATH/Database/BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `users` (`user_institution`, `user_gender`, `user_appellation`, `user_name`, `user_phone`, `user_orcid`, `user_mail`, `user_password`, `created_at`, `updated_at`) VALUES (\'cnd\', \'kadin\', \'Dr.Öğr.Üyesi\', \'gul\', \'+905555435785\', \'1234-5678-9456-1234\', \'gul@gmail.com\', \'$2y$10$i60MPkj6580.0xiCkzdqd.f4WYqGScAJ7jiaHuLQvfLNbAmmCozTa\', \'2025-06-29 21:45:22\', \'2025-06-29 21:45:22\')')
 2 SYSTEMPATH/Database/BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `users` (`user_institution`, `user_gender`, `user_appellation`, `user_name`, `user_phone`, `user_orcid`, `user_mail`, `user_password`, `created_at`, `updated_at`) VALUES (\'cnd\', \'kadin\', \'Dr.Öğr.Üyesi\', \'gul\', \'+905555435785\', \'1234-5678-9456-1234\', \'gul@gmail.com\', \'$2y$10$i60MPkj6580.0xiCkzdqd.f4WYqGScAJ7jiaHuLQvfLNbAmmCozTa\', \'2025-06-29 21:45:22\', \'2025-06-29 21:45:22\')')
 3 SYSTEMPATH/Database/BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`user_institution`, `user_gender`, `user_appellation`, `user_name`, `user_phone`, `user_orcid`, `user_mail`, `user_password`, `created_at`, `updated_at`) VALUES (:user_institution:, :user_gender:, :user_appellation:, :user_name:, :user_phone:, :user_orcid:, :user_mail:, :user_password:, :created_at:, :updated_at:)', [...], false)
 4 SYSTEMPATH/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH/BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH/Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH/Models/LoginModel.php(45): CodeIgniter\Model->insert([...])
 8 APPPATH/Controllers/RegisterController.php(74): App\Models\LoginModel->insertUser([...])
 9 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\RegisterController->submit()
10 SYSTEMPATH/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
11 SYSTEMPATH/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH/Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH/index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-29 21:45:22 --> [Caused by] mysqli_sql_exception: Duplicate entry '' for key 'users.user_username'
in SYSTEMPATH/Database/MySQLi/Connection.php on line 327.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(327): mysqli->query('INSERT INTO `users` (`user_institution`, `user_gender`, `user_appellation`, `user_name`, `user_phone`, `user_orcid`, `user_mail`, `user_password`, `created_at`, `updated_at`) VALUES (\'cnd\', \'kadin\', \'Dr.Öğr.Üyesi\', \'gul\', \'+905555435785\', \'1234-5678-9456-1234\', \'gul@gmail.com\', \'$2y$10$i60MPkj6580.0xiCkzdqd.f4WYqGScAJ7jiaHuLQvfLNbAmmCozTa\', \'2025-06-29 21:45:22\', \'2025-06-29 21:45:22\')', 0)
 2 SYSTEMPATH/Database/BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `users` (`user_institution`, `user_gender`, `user_appellation`, `user_name`, `user_phone`, `user_orcid`, `user_mail`, `user_password`, `created_at`, `updated_at`) VALUES (\'cnd\', \'kadin\', \'Dr.Öğr.Üyesi\', \'gul\', \'+905555435785\', \'1234-5678-9456-1234\', \'gul@gmail.com\', \'$2y$10$i60MPkj6580.0xiCkzdqd.f4WYqGScAJ7jiaHuLQvfLNbAmmCozTa\', \'2025-06-29 21:45:22\', \'2025-06-29 21:45:22\')')
 3 SYSTEMPATH/Database/BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `users` (`user_institution`, `user_gender`, `user_appellation`, `user_name`, `user_phone`, `user_orcid`, `user_mail`, `user_password`, `created_at`, `updated_at`) VALUES (\'cnd\', \'kadin\', \'Dr.Öğr.Üyesi\', \'gul\', \'+905555435785\', \'1234-5678-9456-1234\', \'gul@gmail.com\', \'$2y$10$i60MPkj6580.0xiCkzdqd.f4WYqGScAJ7jiaHuLQvfLNbAmmCozTa\', \'2025-06-29 21:45:22\', \'2025-06-29 21:45:22\')')
 4 SYSTEMPATH/Database/BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`user_institution`, `user_gender`, `user_appellation`, `user_name`, `user_phone`, `user_orcid`, `user_mail`, `user_password`, `created_at`, `updated_at`) VALUES (:user_institution:, :user_gender:, :user_appellation:, :user_name:, :user_phone:, :user_orcid:, :user_mail:, :user_password:, :created_at:, :updated_at:)', [...], false)
 5 SYSTEMPATH/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH/BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH/Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH/Models/LoginModel.php(45): CodeIgniter\Model->insert([...])
 9 APPPATH/Controllers/RegisterController.php(74): App\Models\LoginModel->insertUser([...])
10 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\RegisterController->submit()
11 SYSTEMPATH/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
12 SYSTEMPATH/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 SYSTEMPATH/Boot.php(334): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
15 FCPATH/index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-06-29 21:46:16 --> mysqli_sql_exception: Duplicate entry '' for key 'users.user_username' in /var/www/html/projexiaa/system/Database/MySQLi/Connection.php:327
Stack trace:
#0 /var/www/html/projexiaa/system/Database/MySQLi/Connection.php(327): mysqli->query('INSERT INTO `us...', 0)
#1 /var/www/html/projexiaa/system/Database/BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 /var/www/html/projexiaa/system/Database/BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 /var/www/html/projexiaa/system/Database/BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 /var/www/html/projexiaa/system/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 /var/www/html/projexiaa/system/BaseModel.php(843): CodeIgniter\Model->doInsert(Array)
#6 /var/www/html/projexiaa/system/Model.php(800): CodeIgniter\BaseModel->insert(Array, true)
#7 /var/www/html/projexiaa/app/Models/LoginModel.php(45): CodeIgniter\Model->insert(Array)
#8 /var/www/html/projexiaa/app/Controllers/RegisterController.php(74): App\Models\LoginModel->insertUser(Array)
#9 /var/www/html/projexiaa/system/CodeIgniter.php(933): App\Controllers\RegisterController->submit()
#10 /var/www/html/projexiaa/system/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
#11 /var/www/html/projexiaa/system/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 /var/www/html/projexiaa/system/Boot.php(334): CodeIgniter\CodeIgniter->run()
#13 /var/www/html/projexiaa/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 /var/www/html/projexiaa/index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}
CRITICAL - 2025-06-29 21:46:16 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '' for key 'users.user_username'
[Method: POST, Route: register/submit]
in SYSTEMPATH/Database/BaseConnection.php on line 692.
 1 SYSTEMPATH/Database/BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`user_institution`, `user_gender`, `user_appellation`, `user_name`, `user_phone`, `user_orcid`, `user_mail`, `user_password`, `created_at`, `updated_at`) VALUES (:user_institution:, :user_gender:, :user_appellation:, :user_name:, :user_phone:, :user_orcid:, :user_mail:, :user_password:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH/BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH/Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH/Models/LoginModel.php(45): CodeIgniter\Model->insert([...])
 6 APPPATH/Controllers/RegisterController.php(74): App\Models\LoginModel->insertUser([...])
 7 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\RegisterController->submit()
 8 SYSTEMPATH/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
 9 SYSTEMPATH/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH/Boot.php(334): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH/index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-29 21:46:16 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '' for key 'users.user_username'
in SYSTEMPATH/Database/MySQLi/Connection.php on line 332.
 1 SYSTEMPATH/Database/BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `users` (`user_institution`, `user_gender`, `user_appellation`, `user_name`, `user_phone`, `user_orcid`, `user_mail`, `user_password`, `created_at`, `updated_at`) VALUES (\'cndd\', \'kadin\', \'Dr.Öğr.Üyesi\', \'gul\', \'+905555435785\', \'1234-5678-9456-1234\', \'gul@gmail.com\', \'$2y$10$jXj6LQ6tXwb9nD.eohgMBejZm2A9ARjQTI.IepnDQqt5aeI2ZNcDy\', \'2025-06-29 21:46:16\', \'2025-06-29 21:46:16\')')
 2 SYSTEMPATH/Database/BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `users` (`user_institution`, `user_gender`, `user_appellation`, `user_name`, `user_phone`, `user_orcid`, `user_mail`, `user_password`, `created_at`, `updated_at`) VALUES (\'cndd\', \'kadin\', \'Dr.Öğr.Üyesi\', \'gul\', \'+905555435785\', \'1234-5678-9456-1234\', \'gul@gmail.com\', \'$2y$10$jXj6LQ6tXwb9nD.eohgMBejZm2A9ARjQTI.IepnDQqt5aeI2ZNcDy\', \'2025-06-29 21:46:16\', \'2025-06-29 21:46:16\')')
 3 SYSTEMPATH/Database/BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`user_institution`, `user_gender`, `user_appellation`, `user_name`, `user_phone`, `user_orcid`, `user_mail`, `user_password`, `created_at`, `updated_at`) VALUES (:user_institution:, :user_gender:, :user_appellation:, :user_name:, :user_phone:, :user_orcid:, :user_mail:, :user_password:, :created_at:, :updated_at:)', [...], false)
 4 SYSTEMPATH/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH/BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH/Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH/Models/LoginModel.php(45): CodeIgniter\Model->insert([...])
 8 APPPATH/Controllers/RegisterController.php(74): App\Models\LoginModel->insertUser([...])
 9 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\RegisterController->submit()
10 SYSTEMPATH/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
11 SYSTEMPATH/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH/Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH/index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-29 21:46:16 --> [Caused by] mysqli_sql_exception: Duplicate entry '' for key 'users.user_username'
in SYSTEMPATH/Database/MySQLi/Connection.php on line 327.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(327): mysqli->query('INSERT INTO `users` (`user_institution`, `user_gender`, `user_appellation`, `user_name`, `user_phone`, `user_orcid`, `user_mail`, `user_password`, `created_at`, `updated_at`) VALUES (\'cndd\', \'kadin\', \'Dr.Öğr.Üyesi\', \'gul\', \'+905555435785\', \'1234-5678-9456-1234\', \'gul@gmail.com\', \'$2y$10$jXj6LQ6tXwb9nD.eohgMBejZm2A9ARjQTI.IepnDQqt5aeI2ZNcDy\', \'2025-06-29 21:46:16\', \'2025-06-29 21:46:16\')', 0)
 2 SYSTEMPATH/Database/BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `users` (`user_institution`, `user_gender`, `user_appellation`, `user_name`, `user_phone`, `user_orcid`, `user_mail`, `user_password`, `created_at`, `updated_at`) VALUES (\'cndd\', \'kadin\', \'Dr.Öğr.Üyesi\', \'gul\', \'+905555435785\', \'1234-5678-9456-1234\', \'gul@gmail.com\', \'$2y$10$jXj6LQ6tXwb9nD.eohgMBejZm2A9ARjQTI.IepnDQqt5aeI2ZNcDy\', \'2025-06-29 21:46:16\', \'2025-06-29 21:46:16\')')
 3 SYSTEMPATH/Database/BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `users` (`user_institution`, `user_gender`, `user_appellation`, `user_name`, `user_phone`, `user_orcid`, `user_mail`, `user_password`, `created_at`, `updated_at`) VALUES (\'cndd\', \'kadin\', \'Dr.Öğr.Üyesi\', \'gul\', \'+905555435785\', \'1234-5678-9456-1234\', \'gul@gmail.com\', \'$2y$10$jXj6LQ6tXwb9nD.eohgMBejZm2A9ARjQTI.IepnDQqt5aeI2ZNcDy\', \'2025-06-29 21:46:16\', \'2025-06-29 21:46:16\')')
 4 SYSTEMPATH/Database/BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`user_institution`, `user_gender`, `user_appellation`, `user_name`, `user_phone`, `user_orcid`, `user_mail`, `user_password`, `created_at`, `updated_at`) VALUES (:user_institution:, :user_gender:, :user_appellation:, :user_name:, :user_phone:, :user_orcid:, :user_mail:, :user_password:, :created_at:, :updated_at:)', [...], false)
 5 SYSTEMPATH/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH/BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH/Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH/Models/LoginModel.php(45): CodeIgniter\Model->insert([...])
 9 APPPATH/Controllers/RegisterController.php(74): App\Models\LoginModel->insertUser([...])
10 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\RegisterController->submit()
11 SYSTEMPATH/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
12 SYSTEMPATH/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 SYSTEMPATH/Boot.php(334): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
15 FCPATH/index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-06-29 21:46:25 --> mysqli_sql_exception: Duplicate entry '' for key 'users.user_username' in /var/www/html/projexiaa/system/Database/MySQLi/Connection.php:327
Stack trace:
#0 /var/www/html/projexiaa/system/Database/MySQLi/Connection.php(327): mysqli->query('INSERT INTO `us...', 0)
#1 /var/www/html/projexiaa/system/Database/BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 /var/www/html/projexiaa/system/Database/BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 /var/www/html/projexiaa/system/Database/BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 /var/www/html/projexiaa/system/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 /var/www/html/projexiaa/system/BaseModel.php(843): CodeIgniter\Model->doInsert(Array)
#6 /var/www/html/projexiaa/system/Model.php(800): CodeIgniter\BaseModel->insert(Array, true)
#7 /var/www/html/projexiaa/app/Models/LoginModel.php(45): CodeIgniter\Model->insert(Array)
#8 /var/www/html/projexiaa/app/Controllers/RegisterController.php(74): App\Models\LoginModel->insertUser(Array)
#9 /var/www/html/projexiaa/system/CodeIgniter.php(933): App\Controllers\RegisterController->submit()
#10 /var/www/html/projexiaa/system/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
#11 /var/www/html/projexiaa/system/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 /var/www/html/projexiaa/system/Boot.php(334): CodeIgniter\CodeIgniter->run()
#13 /var/www/html/projexiaa/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 /var/www/html/projexiaa/index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}
CRITICAL - 2025-06-29 21:46:25 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '' for key 'users.user_username'
[Method: POST, Route: register/submit]
in SYSTEMPATH/Database/BaseConnection.php on line 692.
 1 SYSTEMPATH/Database/BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`user_institution`, `user_gender`, `user_appellation`, `user_name`, `user_phone`, `user_orcid`, `user_mail`, `user_password`, `created_at`, `updated_at`) VALUES (:user_institution:, :user_gender:, :user_appellation:, :user_name:, :user_phone:, :user_orcid:, :user_mail:, :user_password:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH/BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH/Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH/Models/LoginModel.php(45): CodeIgniter\Model->insert([...])
 6 APPPATH/Controllers/RegisterController.php(74): App\Models\LoginModel->insertUser([...])
 7 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\RegisterController->submit()
 8 SYSTEMPATH/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
 9 SYSTEMPATH/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH/Boot.php(334): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH/index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-29 21:46:25 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '' for key 'users.user_username'
in SYSTEMPATH/Database/MySQLi/Connection.php on line 332.
 1 SYSTEMPATH/Database/BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `users` (`user_institution`, `user_gender`, `user_appellation`, `user_name`, `user_phone`, `user_orcid`, `user_mail`, `user_password`, `created_at`, `updated_at`) VALUES (\'cndd\', \'kadin\', \'Dr.Öğr.Üyesi\', \'gul\', \'+905555835785\', \'1234-5678-9456-1234\', \'gul@gmail.com\', \'$2y$10$BHrM6Bu4iR2AxYwUKPMIhOGwDvmYY1c/6JAqHIK3DqpD5TUTRcLFK\', \'2025-06-29 21:46:25\', \'2025-06-29 21:46:25\')')
 2 SYSTEMPATH/Database/BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `users` (`user_institution`, `user_gender`, `user_appellation`, `user_name`, `user_phone`, `user_orcid`, `user_mail`, `user_password`, `created_at`, `updated_at`) VALUES (\'cndd\', \'kadin\', \'Dr.Öğr.Üyesi\', \'gul\', \'+905555835785\', \'1234-5678-9456-1234\', \'gul@gmail.com\', \'$2y$10$BHrM6Bu4iR2AxYwUKPMIhOGwDvmYY1c/6JAqHIK3DqpD5TUTRcLFK\', \'2025-06-29 21:46:25\', \'2025-06-29 21:46:25\')')
 3 SYSTEMPATH/Database/BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`user_institution`, `user_gender`, `user_appellation`, `user_name`, `user_phone`, `user_orcid`, `user_mail`, `user_password`, `created_at`, `updated_at`) VALUES (:user_institution:, :user_gender:, :user_appellation:, :user_name:, :user_phone:, :user_orcid:, :user_mail:, :user_password:, :created_at:, :updated_at:)', [...], false)
 4 SYSTEMPATH/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH/BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH/Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH/Models/LoginModel.php(45): CodeIgniter\Model->insert([...])
 8 APPPATH/Controllers/RegisterController.php(74): App\Models\LoginModel->insertUser([...])
 9 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\RegisterController->submit()
10 SYSTEMPATH/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
11 SYSTEMPATH/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH/Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH/index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-29 21:46:25 --> [Caused by] mysqli_sql_exception: Duplicate entry '' for key 'users.user_username'
in SYSTEMPATH/Database/MySQLi/Connection.php on line 327.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(327): mysqli->query('INSERT INTO `users` (`user_institution`, `user_gender`, `user_appellation`, `user_name`, `user_phone`, `user_orcid`, `user_mail`, `user_password`, `created_at`, `updated_at`) VALUES (\'cndd\', \'kadin\', \'Dr.Öğr.Üyesi\', \'gul\', \'+905555835785\', \'1234-5678-9456-1234\', \'gul@gmail.com\', \'$2y$10$BHrM6Bu4iR2AxYwUKPMIhOGwDvmYY1c/6JAqHIK3DqpD5TUTRcLFK\', \'2025-06-29 21:46:25\', \'2025-06-29 21:46:25\')', 0)
 2 SYSTEMPATH/Database/BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `users` (`user_institution`, `user_gender`, `user_appellation`, `user_name`, `user_phone`, `user_orcid`, `user_mail`, `user_password`, `created_at`, `updated_at`) VALUES (\'cndd\', \'kadin\', \'Dr.Öğr.Üyesi\', \'gul\', \'+905555835785\', \'1234-5678-9456-1234\', \'gul@gmail.com\', \'$2y$10$BHrM6Bu4iR2AxYwUKPMIhOGwDvmYY1c/6JAqHIK3DqpD5TUTRcLFK\', \'2025-06-29 21:46:25\', \'2025-06-29 21:46:25\')')
 3 SYSTEMPATH/Database/BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `users` (`user_institution`, `user_gender`, `user_appellation`, `user_name`, `user_phone`, `user_orcid`, `user_mail`, `user_password`, `created_at`, `updated_at`) VALUES (\'cndd\', \'kadin\', \'Dr.Öğr.Üyesi\', \'gul\', \'+905555835785\', \'1234-5678-9456-1234\', \'gul@gmail.com\', \'$2y$10$BHrM6Bu4iR2AxYwUKPMIhOGwDvmYY1c/6JAqHIK3DqpD5TUTRcLFK\', \'2025-06-29 21:46:25\', \'2025-06-29 21:46:25\')')
 4 SYSTEMPATH/Database/BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`user_institution`, `user_gender`, `user_appellation`, `user_name`, `user_phone`, `user_orcid`, `user_mail`, `user_password`, `created_at`, `updated_at`) VALUES (:user_institution:, :user_gender:, :user_appellation:, :user_name:, :user_phone:, :user_orcid:, :user_mail:, :user_password:, :created_at:, :updated_at:)', [...], false)
 5 SYSTEMPATH/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH/BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH/Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH/Models/LoginModel.php(45): CodeIgniter\Model->insert([...])
 9 APPPATH/Controllers/RegisterController.php(74): App\Models\LoginModel->insertUser([...])
10 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\RegisterController->submit()
11 SYSTEMPATH/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
12 SYSTEMPATH/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 SYSTEMPATH/Boot.php(334): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
15 FCPATH/index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-06-29 21:46:45 --> mysqli_sql_exception: Duplicate entry '' for key 'users.user_username' in /var/www/html/projexiaa/system/Database/MySQLi/Connection.php:327
Stack trace:
#0 /var/www/html/projexiaa/system/Database/MySQLi/Connection.php(327): mysqli->query('INSERT INTO `us...', 0)
#1 /var/www/html/projexiaa/system/Database/BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 /var/www/html/projexiaa/system/Database/BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 /var/www/html/projexiaa/system/Database/BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 /var/www/html/projexiaa/system/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 /var/www/html/projexiaa/system/BaseModel.php(843): CodeIgniter\Model->doInsert(Array)
#6 /var/www/html/projexiaa/system/Model.php(800): CodeIgniter\BaseModel->insert(Array, true)
#7 /var/www/html/projexiaa/app/Models/LoginModel.php(45): CodeIgniter\Model->insert(Array)
#8 /var/www/html/projexiaa/app/Controllers/RegisterController.php(74): App\Models\LoginModel->insertUser(Array)
#9 /var/www/html/projexiaa/system/CodeIgniter.php(933): App\Controllers\RegisterController->submit()
#10 /var/www/html/projexiaa/system/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
#11 /var/www/html/projexiaa/system/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 /var/www/html/projexiaa/system/Boot.php(334): CodeIgniter\CodeIgniter->run()
#13 /var/www/html/projexiaa/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 /var/www/html/projexiaa/index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}
CRITICAL - 2025-06-29 21:46:45 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '' for key 'users.user_username'
[Method: POST, Route: register/submit]
in SYSTEMPATH/Database/BaseConnection.php on line 692.
 1 SYSTEMPATH/Database/BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`user_institution`, `user_gender`, `user_appellation`, `user_name`, `user_phone`, `user_orcid`, `user_mail`, `user_password`, `created_at`, `updated_at`) VALUES (:user_institution:, :user_gender:, :user_appellation:, :user_name:, :user_phone:, :user_orcid:, :user_mail:, :user_password:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH/BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH/Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH/Models/LoginModel.php(45): CodeIgniter\Model->insert([...])
 6 APPPATH/Controllers/RegisterController.php(74): App\Models\LoginModel->insertUser([...])
 7 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\RegisterController->submit()
 8 SYSTEMPATH/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
 9 SYSTEMPATH/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH/Boot.php(334): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH/index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-29 21:46:45 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '' for key 'users.user_username'
in SYSTEMPATH/Database/MySQLi/Connection.php on line 332.
 1 SYSTEMPATH/Database/BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `users` (`user_institution`, `user_gender`, `user_appellation`, `user_name`, `user_phone`, `user_orcid`, `user_mail`, `user_password`, `created_at`, `updated_at`) VALUES (\'dadasdsadsa\', \'kadin\', \'Dr.Öğr.Üyesi\', \'gul\', \'+905555835785\', \'1234-5678-9456-1234\', \'gul@gmail.com\', \'$2y$10$lg9IlnZpxkbzxmrgV0gY0OsnbdE1RmhjgGn1vcEeWO1CGGdAMiscq\', \'2025-06-29 21:46:45\', \'2025-06-29 21:46:45\')')
 2 SYSTEMPATH/Database/BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `users` (`user_institution`, `user_gender`, `user_appellation`, `user_name`, `user_phone`, `user_orcid`, `user_mail`, `user_password`, `created_at`, `updated_at`) VALUES (\'dadasdsadsa\', \'kadin\', \'Dr.Öğr.Üyesi\', \'gul\', \'+905555835785\', \'1234-5678-9456-1234\', \'gul@gmail.com\', \'$2y$10$lg9IlnZpxkbzxmrgV0gY0OsnbdE1RmhjgGn1vcEeWO1CGGdAMiscq\', \'2025-06-29 21:46:45\', \'2025-06-29 21:46:45\')')
 3 SYSTEMPATH/Database/BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`user_institution`, `user_gender`, `user_appellation`, `user_name`, `user_phone`, `user_orcid`, `user_mail`, `user_password`, `created_at`, `updated_at`) VALUES (:user_institution:, :user_gender:, :user_appellation:, :user_name:, :user_phone:, :user_orcid:, :user_mail:, :user_password:, :created_at:, :updated_at:)', [...], false)
 4 SYSTEMPATH/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH/BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH/Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH/Models/LoginModel.php(45): CodeIgniter\Model->insert([...])
 8 APPPATH/Controllers/RegisterController.php(74): App\Models\LoginModel->insertUser([...])
 9 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\RegisterController->submit()
10 SYSTEMPATH/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
11 SYSTEMPATH/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH/Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH/index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-29 21:46:45 --> [Caused by] mysqli_sql_exception: Duplicate entry '' for key 'users.user_username'
in SYSTEMPATH/Database/MySQLi/Connection.php on line 327.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(327): mysqli->query('INSERT INTO `users` (`user_institution`, `user_gender`, `user_appellation`, `user_name`, `user_phone`, `user_orcid`, `user_mail`, `user_password`, `created_at`, `updated_at`) VALUES (\'dadasdsadsa\', \'kadin\', \'Dr.Öğr.Üyesi\', \'gul\', \'+905555835785\', \'1234-5678-9456-1234\', \'gul@gmail.com\', \'$2y$10$lg9IlnZpxkbzxmrgV0gY0OsnbdE1RmhjgGn1vcEeWO1CGGdAMiscq\', \'2025-06-29 21:46:45\', \'2025-06-29 21:46:45\')', 0)
 2 SYSTEMPATH/Database/BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `users` (`user_institution`, `user_gender`, `user_appellation`, `user_name`, `user_phone`, `user_orcid`, `user_mail`, `user_password`, `created_at`, `updated_at`) VALUES (\'dadasdsadsa\', \'kadin\', \'Dr.Öğr.Üyesi\', \'gul\', \'+905555835785\', \'1234-5678-9456-1234\', \'gul@gmail.com\', \'$2y$10$lg9IlnZpxkbzxmrgV0gY0OsnbdE1RmhjgGn1vcEeWO1CGGdAMiscq\', \'2025-06-29 21:46:45\', \'2025-06-29 21:46:45\')')
 3 SYSTEMPATH/Database/BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `users` (`user_institution`, `user_gender`, `user_appellation`, `user_name`, `user_phone`, `user_orcid`, `user_mail`, `user_password`, `created_at`, `updated_at`) VALUES (\'dadasdsadsa\', \'kadin\', \'Dr.Öğr.Üyesi\', \'gul\', \'+905555835785\', \'1234-5678-9456-1234\', \'gul@gmail.com\', \'$2y$10$lg9IlnZpxkbzxmrgV0gY0OsnbdE1RmhjgGn1vcEeWO1CGGdAMiscq\', \'2025-06-29 21:46:45\', \'2025-06-29 21:46:45\')')
 4 SYSTEMPATH/Database/BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`user_institution`, `user_gender`, `user_appellation`, `user_name`, `user_phone`, `user_orcid`, `user_mail`, `user_password`, `created_at`, `updated_at`) VALUES (:user_institution:, :user_gender:, :user_appellation:, :user_name:, :user_phone:, :user_orcid:, :user_mail:, :user_password:, :created_at:, :updated_at:)', [...], false)
 5 SYSTEMPATH/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH/BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH/Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH/Models/LoginModel.php(45): CodeIgniter\Model->insert([...])
 9 APPPATH/Controllers/RegisterController.php(74): App\Models\LoginModel->insertUser([...])
10 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\RegisterController->submit()
11 SYSTEMPATH/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
12 SYSTEMPATH/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 SYSTEMPATH/Boot.php(334): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
15 FCPATH/index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-06-29 21:47:05 --> mysqli_sql_exception: Duplicate entry '' for key 'users.user_username' in /var/www/html/projexiaa/system/Database/MySQLi/Connection.php:327
Stack trace:
#0 /var/www/html/projexiaa/system/Database/MySQLi/Connection.php(327): mysqli->query('INSERT INTO `us...', 0)
#1 /var/www/html/projexiaa/system/Database/BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 /var/www/html/projexiaa/system/Database/BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 /var/www/html/projexiaa/system/Database/BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 /var/www/html/projexiaa/system/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 /var/www/html/projexiaa/system/BaseModel.php(843): CodeIgniter\Model->doInsert(Array)
#6 /var/www/html/projexiaa/system/Model.php(800): CodeIgniter\BaseModel->insert(Array, true)
#7 /var/www/html/projexiaa/app/Models/LoginModel.php(45): CodeIgniter\Model->insert(Array)
#8 /var/www/html/projexiaa/app/Controllers/RegisterController.php(74): App\Models\LoginModel->insertUser(Array)
#9 /var/www/html/projexiaa/system/CodeIgniter.php(933): App\Controllers\RegisterController->submit()
#10 /var/www/html/projexiaa/system/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
#11 /var/www/html/projexiaa/system/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 /var/www/html/projexiaa/system/Boot.php(334): CodeIgniter\CodeIgniter->run()
#13 /var/www/html/projexiaa/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 /var/www/html/projexiaa/index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}
CRITICAL - 2025-06-29 21:47:05 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '' for key 'users.user_username'
[Method: POST, Route: register/submit]
in SYSTEMPATH/Database/BaseConnection.php on line 692.
 1 SYSTEMPATH/Database/BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`user_institution`, `user_gender`, `user_appellation`, `user_name`, `user_phone`, `user_orcid`, `user_mail`, `user_password`, `created_at`, `updated_at`) VALUES (:user_institution:, :user_gender:, :user_appellation:, :user_name:, :user_phone:, :user_orcid:, :user_mail:, :user_password:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH/BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH/Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH/Models/LoginModel.php(45): CodeIgniter\Model->insert([...])
 6 APPPATH/Controllers/RegisterController.php(74): App\Models\LoginModel->insertUser([...])
 7 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\RegisterController->submit()
 8 SYSTEMPATH/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
 9 SYSTEMPATH/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH/Boot.php(334): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH/index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-29 21:47:05 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '' for key 'users.user_username'
in SYSTEMPATH/Database/MySQLi/Connection.php on line 332.
 1 SYSTEMPATH/Database/BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `users` (`user_institution`, `user_gender`, `user_appellation`, `user_name`, `user_phone`, `user_orcid`, `user_mail`, `user_password`, `created_at`, `updated_at`) VALUES (\'dadasdsadsa\', \'kadin\', \'Dr.Öğr.Üyesi\', \'gul\', \'+905555835785\', \'1234-5678-9456-1234\', \'gul@gmail.com\', \'$2y$10$DmAAv8Jf5OiAS4b5bHPsbe9pC92ZhSW0Z.z5sdWAYaCED5GLdPDKq\', \'2025-06-29 21:47:05\', \'2025-06-29 21:47:05\')')
 2 SYSTEMPATH/Database/BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `users` (`user_institution`, `user_gender`, `user_appellation`, `user_name`, `user_phone`, `user_orcid`, `user_mail`, `user_password`, `created_at`, `updated_at`) VALUES (\'dadasdsadsa\', \'kadin\', \'Dr.Öğr.Üyesi\', \'gul\', \'+905555835785\', \'1234-5678-9456-1234\', \'gul@gmail.com\', \'$2y$10$DmAAv8Jf5OiAS4b5bHPsbe9pC92ZhSW0Z.z5sdWAYaCED5GLdPDKq\', \'2025-06-29 21:47:05\', \'2025-06-29 21:47:05\')')
 3 SYSTEMPATH/Database/BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`user_institution`, `user_gender`, `user_appellation`, `user_name`, `user_phone`, `user_orcid`, `user_mail`, `user_password`, `created_at`, `updated_at`) VALUES (:user_institution:, :user_gender:, :user_appellation:, :user_name:, :user_phone:, :user_orcid:, :user_mail:, :user_password:, :created_at:, :updated_at:)', [...], false)
 4 SYSTEMPATH/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH/BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH/Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH/Models/LoginModel.php(45): CodeIgniter\Model->insert([...])
 8 APPPATH/Controllers/RegisterController.php(74): App\Models\LoginModel->insertUser([...])
 9 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\RegisterController->submit()
10 SYSTEMPATH/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
11 SYSTEMPATH/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH/Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH/index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-29 21:47:05 --> [Caused by] mysqli_sql_exception: Duplicate entry '' for key 'users.user_username'
in SYSTEMPATH/Database/MySQLi/Connection.php on line 327.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(327): mysqli->query('INSERT INTO `users` (`user_institution`, `user_gender`, `user_appellation`, `user_name`, `user_phone`, `user_orcid`, `user_mail`, `user_password`, `created_at`, `updated_at`) VALUES (\'dadasdsadsa\', \'kadin\', \'Dr.Öğr.Üyesi\', \'gul\', \'+905555835785\', \'1234-5678-9456-1234\', \'gul@gmail.com\', \'$2y$10$DmAAv8Jf5OiAS4b5bHPsbe9pC92ZhSW0Z.z5sdWAYaCED5GLdPDKq\', \'2025-06-29 21:47:05\', \'2025-06-29 21:47:05\')', 0)
 2 SYSTEMPATH/Database/BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `users` (`user_institution`, `user_gender`, `user_appellation`, `user_name`, `user_phone`, `user_orcid`, `user_mail`, `user_password`, `created_at`, `updated_at`) VALUES (\'dadasdsadsa\', \'kadin\', \'Dr.Öğr.Üyesi\', \'gul\', \'+905555835785\', \'1234-5678-9456-1234\', \'gul@gmail.com\', \'$2y$10$DmAAv8Jf5OiAS4b5bHPsbe9pC92ZhSW0Z.z5sdWAYaCED5GLdPDKq\', \'2025-06-29 21:47:05\', \'2025-06-29 21:47:05\')')
 3 SYSTEMPATH/Database/BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `users` (`user_institution`, `user_gender`, `user_appellation`, `user_name`, `user_phone`, `user_orcid`, `user_mail`, `user_password`, `created_at`, `updated_at`) VALUES (\'dadasdsadsa\', \'kadin\', \'Dr.Öğr.Üyesi\', \'gul\', \'+905555835785\', \'1234-5678-9456-1234\', \'gul@gmail.com\', \'$2y$10$DmAAv8Jf5OiAS4b5bHPsbe9pC92ZhSW0Z.z5sdWAYaCED5GLdPDKq\', \'2025-06-29 21:47:05\', \'2025-06-29 21:47:05\')')
 4 SYSTEMPATH/Database/BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`user_institution`, `user_gender`, `user_appellation`, `user_name`, `user_phone`, `user_orcid`, `user_mail`, `user_password`, `created_at`, `updated_at`) VALUES (:user_institution:, :user_gender:, :user_appellation:, :user_name:, :user_phone:, :user_orcid:, :user_mail:, :user_password:, :created_at:, :updated_at:)', [...], false)
 5 SYSTEMPATH/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH/BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH/Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH/Models/LoginModel.php(45): CodeIgniter\Model->insert([...])
 9 APPPATH/Controllers/RegisterController.php(74): App\Models\LoginModel->insertUser([...])
10 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\RegisterController->submit()
11 SYSTEMPATH/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
12 SYSTEMPATH/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 SYSTEMPATH/Boot.php(334): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
15 FCPATH/index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-06-29 21:47:55 --> mysqli_sql_exception: Duplicate entry '' for key 'users.user_username' in /var/www/html/projexiaa/system/Database/MySQLi/Connection.php:327
Stack trace:
#0 /var/www/html/projexiaa/system/Database/MySQLi/Connection.php(327): mysqli->query('INSERT INTO `us...', 0)
#1 /var/www/html/projexiaa/system/Database/BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 /var/www/html/projexiaa/system/Database/BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 /var/www/html/projexiaa/system/Database/BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 /var/www/html/projexiaa/system/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 /var/www/html/projexiaa/system/BaseModel.php(843): CodeIgniter\Model->doInsert(Array)
#6 /var/www/html/projexiaa/system/Model.php(800): CodeIgniter\BaseModel->insert(Array, true)
#7 /var/www/html/projexiaa/app/Models/LoginModel.php(45): CodeIgniter\Model->insert(Array)
#8 /var/www/html/projexiaa/app/Controllers/RegisterController.php(74): App\Models\LoginModel->insertUser(Array)
#9 /var/www/html/projexiaa/system/CodeIgniter.php(933): App\Controllers\RegisterController->submit()
#10 /var/www/html/projexiaa/system/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
#11 /var/www/html/projexiaa/system/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 /var/www/html/projexiaa/system/Boot.php(334): CodeIgniter\CodeIgniter->run()
#13 /var/www/html/projexiaa/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 /var/www/html/projexiaa/index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}
CRITICAL - 2025-06-29 21:47:55 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '' for key 'users.user_username'
[Method: POST, Route: register/submit]
in SYSTEMPATH/Database/BaseConnection.php on line 692.
 1 SYSTEMPATH/Database/BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`user_institution`, `user_gender`, `user_appellation`, `user_name`, `user_phone`, `user_orcid`, `user_mail`, `user_password`, `created_at`, `updated_at`) VALUES (:user_institution:, :user_gender:, :user_appellation:, :user_name:, :user_phone:, :user_orcid:, :user_mail:, :user_password:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH/BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH/Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH/Models/LoginModel.php(45): CodeIgniter\Model->insert([...])
 6 APPPATH/Controllers/RegisterController.php(74): App\Models\LoginModel->insertUser([...])
 7 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\RegisterController->submit()
 8 SYSTEMPATH/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
 9 SYSTEMPATH/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH/Boot.php(334): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH/index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-29 21:47:55 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '' for key 'users.user_username'
in SYSTEMPATH/Database/MySQLi/Connection.php on line 332.
 1 SYSTEMPATH/Database/BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `users` (`user_institution`, `user_gender`, `user_appellation`, `user_name`, `user_phone`, `user_orcid`, `user_mail`, `user_password`, `created_at`, `updated_at`) VALUES (\'dadasdsadsa\', \'kadin\', \'Dr.Öğr.Üyesi\', \'gul dhdahsjgghdsa\', \'+905555835785\', \'1234-5678-9456-1234\', \'gul@gmail.com\', \'$2y$10$OEzFCA2MPqspQDjiDDI0K.fG0OStfCDWqVi/fFc3eCJTNzyXBy7mG\', \'2025-06-29 21:47:55\', \'2025-06-29 21:47:55\')')
 2 SYSTEMPATH/Database/BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `users` (`user_institution`, `user_gender`, `user_appellation`, `user_name`, `user_phone`, `user_orcid`, `user_mail`, `user_password`, `created_at`, `updated_at`) VALUES (\'dadasdsadsa\', \'kadin\', \'Dr.Öğr.Üyesi\', \'gul dhdahsjgghdsa\', \'+905555835785\', \'1234-5678-9456-1234\', \'gul@gmail.com\', \'$2y$10$OEzFCA2MPqspQDjiDDI0K.fG0OStfCDWqVi/fFc3eCJTNzyXBy7mG\', \'2025-06-29 21:47:55\', \'2025-06-29 21:47:55\')')
 3 SYSTEMPATH/Database/BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`user_institution`, `user_gender`, `user_appellation`, `user_name`, `user_phone`, `user_orcid`, `user_mail`, `user_password`, `created_at`, `updated_at`) VALUES (:user_institution:, :user_gender:, :user_appellation:, :user_name:, :user_phone:, :user_orcid:, :user_mail:, :user_password:, :created_at:, :updated_at:)', [...], false)
 4 SYSTEMPATH/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH/BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH/Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH/Models/LoginModel.php(45): CodeIgniter\Model->insert([...])
 8 APPPATH/Controllers/RegisterController.php(74): App\Models\LoginModel->insertUser([...])
 9 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\RegisterController->submit()
10 SYSTEMPATH/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
11 SYSTEMPATH/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH/Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH/index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-29 21:47:55 --> [Caused by] mysqli_sql_exception: Duplicate entry '' for key 'users.user_username'
in SYSTEMPATH/Database/MySQLi/Connection.php on line 327.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(327): mysqli->query('INSERT INTO `users` (`user_institution`, `user_gender`, `user_appellation`, `user_name`, `user_phone`, `user_orcid`, `user_mail`, `user_password`, `created_at`, `updated_at`) VALUES (\'dadasdsadsa\', \'kadin\', \'Dr.Öğr.Üyesi\', \'gul dhdahsjgghdsa\', \'+905555835785\', \'1234-5678-9456-1234\', \'gul@gmail.com\', \'$2y$10$OEzFCA2MPqspQDjiDDI0K.fG0OStfCDWqVi/fFc3eCJTNzyXBy7mG\', \'2025-06-29 21:47:55\', \'2025-06-29 21:47:55\')', 0)
 2 SYSTEMPATH/Database/BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `users` (`user_institution`, `user_gender`, `user_appellation`, `user_name`, `user_phone`, `user_orcid`, `user_mail`, `user_password`, `created_at`, `updated_at`) VALUES (\'dadasdsadsa\', \'kadin\', \'Dr.Öğr.Üyesi\', \'gul dhdahsjgghdsa\', \'+905555835785\', \'1234-5678-9456-1234\', \'gul@gmail.com\', \'$2y$10$OEzFCA2MPqspQDjiDDI0K.fG0OStfCDWqVi/fFc3eCJTNzyXBy7mG\', \'2025-06-29 21:47:55\', \'2025-06-29 21:47:55\')')
 3 SYSTEMPATH/Database/BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `users` (`user_institution`, `user_gender`, `user_appellation`, `user_name`, `user_phone`, `user_orcid`, `user_mail`, `user_password`, `created_at`, `updated_at`) VALUES (\'dadasdsadsa\', \'kadin\', \'Dr.Öğr.Üyesi\', \'gul dhdahsjgghdsa\', \'+905555835785\', \'1234-5678-9456-1234\', \'gul@gmail.com\', \'$2y$10$OEzFCA2MPqspQDjiDDI0K.fG0OStfCDWqVi/fFc3eCJTNzyXBy7mG\', \'2025-06-29 21:47:55\', \'2025-06-29 21:47:55\')')
 4 SYSTEMPATH/Database/BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`user_institution`, `user_gender`, `user_appellation`, `user_name`, `user_phone`, `user_orcid`, `user_mail`, `user_password`, `created_at`, `updated_at`) VALUES (:user_institution:, :user_gender:, :user_appellation:, :user_name:, :user_phone:, :user_orcid:, :user_mail:, :user_password:, :created_at:, :updated_at:)', [...], false)
 5 SYSTEMPATH/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH/BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH/Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH/Models/LoginModel.php(45): CodeIgniter\Model->insert([...])
 9 APPPATH/Controllers/RegisterController.php(74): App\Models\LoginModel->insertUser([...])
10 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\RegisterController->submit()
11 SYSTEMPATH/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
12 SYSTEMPATH/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 SYSTEMPATH/Boot.php(334): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
15 FCPATH/index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-06-29 21:48:00 --> mysqli_sql_exception: Duplicate entry '' for key 'users.user_username' in /var/www/html/projexiaa/system/Database/MySQLi/Connection.php:327
Stack trace:
#0 /var/www/html/projexiaa/system/Database/MySQLi/Connection.php(327): mysqli->query('INSERT INTO `us...', 0)
#1 /var/www/html/projexiaa/system/Database/BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 /var/www/html/projexiaa/system/Database/BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 /var/www/html/projexiaa/system/Database/BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 /var/www/html/projexiaa/system/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 /var/www/html/projexiaa/system/BaseModel.php(843): CodeIgniter\Model->doInsert(Array)
#6 /var/www/html/projexiaa/system/Model.php(800): CodeIgniter\BaseModel->insert(Array, true)
#7 /var/www/html/projexiaa/app/Models/LoginModel.php(45): CodeIgniter\Model->insert(Array)
#8 /var/www/html/projexiaa/app/Controllers/RegisterController.php(74): App\Models\LoginModel->insertUser(Array)
#9 /var/www/html/projexiaa/system/CodeIgniter.php(933): App\Controllers\RegisterController->submit()
#10 /var/www/html/projexiaa/system/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
#11 /var/www/html/projexiaa/system/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 /var/www/html/projexiaa/system/Boot.php(334): CodeIgniter\CodeIgniter->run()
#13 /var/www/html/projexiaa/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 /var/www/html/projexiaa/index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}
CRITICAL - 2025-06-29 21:48:00 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '' for key 'users.user_username'
[Method: POST, Route: register/submit]
in SYSTEMPATH/Database/BaseConnection.php on line 692.
 1 SYSTEMPATH/Database/BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`user_institution`, `user_gender`, `user_appellation`, `user_name`, `user_phone`, `user_orcid`, `user_mail`, `user_password`, `created_at`, `updated_at`) VALUES (:user_institution:, :user_gender:, :user_appellation:, :user_name:, :user_phone:, :user_orcid:, :user_mail:, :user_password:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH/BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH/Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH/Models/LoginModel.php(45): CodeIgniter\Model->insert([...])
 6 APPPATH/Controllers/RegisterController.php(74): App\Models\LoginModel->insertUser([...])
 7 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\RegisterController->submit()
 8 SYSTEMPATH/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
 9 SYSTEMPATH/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH/Boot.php(334): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH/index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-29 21:48:00 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '' for key 'users.user_username'
in SYSTEMPATH/Database/MySQLi/Connection.php on line 332.
 1 SYSTEMPATH/Database/BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `users` (`user_institution`, `user_gender`, `user_appellation`, `user_name`, `user_phone`, `user_orcid`, `user_mail`, `user_password`, `created_at`, `updated_at`) VALUES (\'dadasdsadsa\', \'erkek\', \'Dr.Öğr.Üyesi\', \'gul dhdahsjgghdsa\', \'+905555835785\', \'1234-5678-9456-1234\', \'gul@gmail.com\', \'$2y$10$d7d34yRyTyn//mOAso5VROiAUa1AKqIqn3yO8ib1MFW6E4v592tRy\', \'2025-06-29 21:48:00\', \'2025-06-29 21:48:00\')')
 2 SYSTEMPATH/Database/BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `users` (`user_institution`, `user_gender`, `user_appellation`, `user_name`, `user_phone`, `user_orcid`, `user_mail`, `user_password`, `created_at`, `updated_at`) VALUES (\'dadasdsadsa\', \'erkek\', \'Dr.Öğr.Üyesi\', \'gul dhdahsjgghdsa\', \'+905555835785\', \'1234-5678-9456-1234\', \'gul@gmail.com\', \'$2y$10$d7d34yRyTyn//mOAso5VROiAUa1AKqIqn3yO8ib1MFW6E4v592tRy\', \'2025-06-29 21:48:00\', \'2025-06-29 21:48:00\')')
 3 SYSTEMPATH/Database/BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`user_institution`, `user_gender`, `user_appellation`, `user_name`, `user_phone`, `user_orcid`, `user_mail`, `user_password`, `created_at`, `updated_at`) VALUES (:user_institution:, :user_gender:, :user_appellation:, :user_name:, :user_phone:, :user_orcid:, :user_mail:, :user_password:, :created_at:, :updated_at:)', [...], false)
 4 SYSTEMPATH/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH/BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH/Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH/Models/LoginModel.php(45): CodeIgniter\Model->insert([...])
 8 APPPATH/Controllers/RegisterController.php(74): App\Models\LoginModel->insertUser([...])
 9 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\RegisterController->submit()
10 SYSTEMPATH/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
11 SYSTEMPATH/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH/Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH/index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-29 21:48:00 --> [Caused by] mysqli_sql_exception: Duplicate entry '' for key 'users.user_username'
in SYSTEMPATH/Database/MySQLi/Connection.php on line 327.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(327): mysqli->query('INSERT INTO `users` (`user_institution`, `user_gender`, `user_appellation`, `user_name`, `user_phone`, `user_orcid`, `user_mail`, `user_password`, `created_at`, `updated_at`) VALUES (\'dadasdsadsa\', \'erkek\', \'Dr.Öğr.Üyesi\', \'gul dhdahsjgghdsa\', \'+905555835785\', \'1234-5678-9456-1234\', \'gul@gmail.com\', \'$2y$10$d7d34yRyTyn//mOAso5VROiAUa1AKqIqn3yO8ib1MFW6E4v592tRy\', \'2025-06-29 21:48:00\', \'2025-06-29 21:48:00\')', 0)
 2 SYSTEMPATH/Database/BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `users` (`user_institution`, `user_gender`, `user_appellation`, `user_name`, `user_phone`, `user_orcid`, `user_mail`, `user_password`, `created_at`, `updated_at`) VALUES (\'dadasdsadsa\', \'erkek\', \'Dr.Öğr.Üyesi\', \'gul dhdahsjgghdsa\', \'+905555835785\', \'1234-5678-9456-1234\', \'gul@gmail.com\', \'$2y$10$d7d34yRyTyn//mOAso5VROiAUa1AKqIqn3yO8ib1MFW6E4v592tRy\', \'2025-06-29 21:48:00\', \'2025-06-29 21:48:00\')')
 3 SYSTEMPATH/Database/BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `users` (`user_institution`, `user_gender`, `user_appellation`, `user_name`, `user_phone`, `user_orcid`, `user_mail`, `user_password`, `created_at`, `updated_at`) VALUES (\'dadasdsadsa\', \'erkek\', \'Dr.Öğr.Üyesi\', \'gul dhdahsjgghdsa\', \'+905555835785\', \'1234-5678-9456-1234\', \'gul@gmail.com\', \'$2y$10$d7d34yRyTyn//mOAso5VROiAUa1AKqIqn3yO8ib1MFW6E4v592tRy\', \'2025-06-29 21:48:00\', \'2025-06-29 21:48:00\')')
 4 SYSTEMPATH/Database/BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`user_institution`, `user_gender`, `user_appellation`, `user_name`, `user_phone`, `user_orcid`, `user_mail`, `user_password`, `created_at`, `updated_at`) VALUES (:user_institution:, :user_gender:, :user_appellation:, :user_name:, :user_phone:, :user_orcid:, :user_mail:, :user_password:, :created_at:, :updated_at:)', [...], false)
 5 SYSTEMPATH/Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH/BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH/Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH/Models/LoginModel.php(45): CodeIgniter\Model->insert([...])
 9 APPPATH/Controllers/RegisterController.php(74): App\Models\LoginModel->insertUser([...])
10 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\RegisterController->submit()
11 SYSTEMPATH/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
12 SYSTEMPATH/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 SYSTEMPATH/Boot.php(334): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
15 FCPATH/index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-06-29 21:49:45 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 21:49:47 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 21:49:57 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 21:49:57 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 21:51:20 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 21:51:26 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 21:52:16 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 21:52:17 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 21:52:25 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 21:52:29 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 21:52:29 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 21:52:37 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 21:52:37 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 21:52:41 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 21:52:43 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 21:52:45 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-06-29 21:52:45 --> ErrorException: Undefined array key "role_id"
[Method: GET, Route: admin/projects/detay/2]
in APPPATH/Models/RoleModel.php on line 68.
 1 APPPATH/Models/RoleModel.php(68): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "role_id"', '/var/www/html/projexiaa/app/Models/RoleModel.php', 68)
 2 APPPATH/Controllers/Admin/ProjectController.php(69): App\Models\RoleModel->getRoleIdByUserId('1')
 3 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Admin\ProjectController->projectdetay('2')
 4 SYSTEMPATH/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\ProjectController))
 5 SYSTEMPATH/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH/Boot.php(334): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH/index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-06-29 21:54:03 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-06-29 21:54:03 --> ErrorException: Undefined array key "role_id"
[Method: GET, Route: admin/projects/detay/2]
in APPPATH/Models/RoleModel.php on line 68.
 1 APPPATH/Models/RoleModel.php(68): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "role_id"', '/var/www/html/projexiaa/app/Models/RoleModel.php', 68)
 2 APPPATH/Controllers/Admin/ProjectController.php(69): App\Models\RoleModel->getRoleIdByUserId('1')
 3 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Admin\ProjectController->projectdetay('2')
 4 SYSTEMPATH/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\ProjectController))
 5 SYSTEMPATH/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH/Boot.php(334): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH/index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-06-29 21:54:03 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-06-29 21:54:03 --> ErrorException: Undefined array key "role_id"
[Method: GET, Route: admin/projects/detay/2]
in APPPATH/Models/RoleModel.php on line 68.
 1 APPPATH/Models/RoleModel.php(68): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "role_id"', '/var/www/html/projexiaa/app/Models/RoleModel.php', 68)
 2 APPPATH/Controllers/Admin/ProjectController.php(69): App\Models\RoleModel->getRoleIdByUserId('1')
 3 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Admin\ProjectController->projectdetay('2')
 4 SYSTEMPATH/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\ProjectController))
 5 SYSTEMPATH/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH/Boot.php(334): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH/index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-06-29 21:55:25 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 21:55:34 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 21:55:38 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 21:55:39 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 21:55:45 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 22:06:10 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 22:06:18 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 22:06:23 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 22:10:23 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-06-29 22:10:23 --> ErrorException: Undefined array key "id"
[Method: GET, Route: user/projects/detay/2]
in APPPATH/Views/users/projectDetail.php on line 213.
 1 APPPATH/Views/users/projectDetail.php(213): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "id"', '/var/www/html/projexiaa/app/Views/users/projectDetail.php', 213)
 2 SYSTEMPATH/View/View.php(224): include('/var/www/html/projexiaa/app/Views/users/projectDetail.php')
 3 SYSTEMPATH/View/View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH/Common.php(1173): CodeIgniter\View\View->render('users/projectDetail', [], true)
 5 APPPATH/Controllers/User/ProjectController.php(411): view('users/projectDetail', [...])
 6 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\User\ProjectController->projectdetay('2')
 7 SYSTEMPATH/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User\ProjectController))
 8 SYSTEMPATH/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH/Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH/index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-06-29 22:10:39 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 22:11:40 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 22:11:43 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 22:11:45 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 22:12:56 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 22:12:57 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-06-29 22:12:57 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "users/projectDuzenle.php"
[Method: GET, Route: duzenle/2]
in SYSTEMPATH/Exceptions/FrameworkException.php on line 39.
 1 SYSTEMPATH/View/View.php(212): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('users/projectDuzenle.php')
 2 SYSTEMPATH/Common.php(1173): CodeIgniter\View\View->render('users/projectDuzenle', [], true)
 3 APPPATH/Controllers/User/ProjectController.php(177): view('users/projectDuzenle', [...])
 4 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\User\ProjectController->edit('2')
 5 SYSTEMPATH/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User\ProjectController))
 6 SYSTEMPATH/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH/Boot.php(334): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH/index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-06-29 22:13:25 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 22:16:38 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 22:16:40 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 22:17:20 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-06-29 22:17:20 --> ErrorException: Undefined array key "id"
[Method: GET, Route: duzenle/2]
in APPPATH/Views/users/projeDuzenle.php on line 41.
 1 APPPATH/Views/users/projeDuzenle.php(41): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "id"', '/var/www/html/projexiaa/app/Views/users/projeDuzenle.php', 41)
 2 SYSTEMPATH/View/View.php(224): include('/var/www/html/projexiaa/app/Views/users/projeDuzenle.php')
 3 SYSTEMPATH/View/View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH/Common.php(1173): CodeIgniter\View\View->render('users/projeDuzenle', [], true)
 5 APPPATH/Controllers/User/ProjectController.php(177): view('users/projeDuzenle', [...])
 6 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\User\ProjectController->edit('2')
 7 SYSTEMPATH/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User\ProjectController))
 8 SYSTEMPATH/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH/Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH/index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-06-29 22:18:04 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 22:19:49 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 22:24:01 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 22:24:41 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 22:24:50 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 22:24:51 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 22:24:55 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 22:24:57 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 22:25:11 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 22:25:13 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 22:34:23 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 22:34:25 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 22:44:11 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 22:44:33 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 22:44:34 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 22:45:54 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 22:45:59 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 22:46:01 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 22:46:40 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 23:04:22 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 23:05:10 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 23:07:27 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 23:07:28 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 23:07:51 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 23:07:52 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 23:08:37 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 23:08:40 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 23:08:41 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 23:09:53 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 23:10:00 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 23:10:01 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 23:10:06 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 23:10:08 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 23:10:33 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 23:13:36 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 23:13:40 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 23:13:42 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 23:13:44 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 23:13:50 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 23:13:53 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 23:13:56 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 23:14:02 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 23:14:05 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 23:14:07 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 23:16:04 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 23:20:05 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 23:20:08 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 23:20:17 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 23:20:19 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 23:20:20 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 23:20:23 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 23:20:27 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 23:20:29 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 23:22:11 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 23:22:14 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 23:22:15 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 23:23:00 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 23:23:07 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 23:23:07 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 23:23:09 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 23:25:31 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 23:25:32 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 23:25:35 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 23:25:36 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 23:25:38 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 23:25:40 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 23:25:48 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 23:26:31 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 23:26:33 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 23:26:36 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 23:27:09 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 23:27:13 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 23:27:49 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 23:27:51 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-29 23:27:53 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
